helm/cert-operator/templates/deployment.yaml

```diff
@@ -39,17 +39,17 @@ spec:
       - name: ssl-certs
         hostPath:
           path: /etc/ssl/certs/
-      serviceAccountName: cert-operator
+      serviceAccountName: {{ include "resource.default.name" . }}
       securityContext:
         runAsUser: {{ .Values.userID }}
         runAsGroup: {{ .Values.groupID }}
       containers:
       - name: cert-operator
-        image: "{{ .Values.Installation.V1.Registry.Domain }}/giantswarm/cert-operator:[[ .SHA ]]"
+        image: "{{ .Values.Installation.V1.Registry.Domain }}/giantswarm/cert-operator:[[ .Version ]]"
         volumeMounts:
-        - name: cert-operator-configmap
+        - name: {{ .Chart.Name }}-configmap
```

Comment:
Wouldn't it be better to `{{ include "name" . }}` here?

Reply:
`"name"` is a bit nicer `.Chart.Name`, so sure.